src/App/Osu/OsuDatabaseBeatmap.cpp

```diff
@@ -1900,12 +1908,26 @@ OsuDatabaseBeatmapStarCalculator::OsuDatabaseBeatmapStarCalculator() : Resource(
 	m_iMaxPossibleCombo = 0;
 }
 
+
+
+
+
+
 void OsuDatabaseBeatmapStarCalculator::init()
 {
 	// NOTE: this accesses runtime mods, so must be run sync (not async)
 	// technically the getSelectedBeatmap() call here is a bit unsafe, since the beatmap could have changed already between async and sync, but in that case we recalculate immediately after anyways
 	if (!m_bDead.load() && m_iErrorCode == 0 && m_diff2->m_osu->getSelectedBeatmap() != NULL)
-		m_pp = OsuDifficultyCalculator::calculatePPv2(m_diff2->m_osu, m_diff2->m_osu->getSelectedBeatmap(), m_aimStars.load(), m_aimSliderFactor.load(), m_speedStars.load(), m_speedNotes.load(), m_iNumObjects.load(), m_iNumCircles.load(), m_iNumSliders.load(), m_iNumSpinners.load(), m_iMaxPossibleCombo);
+		if (osu_map_pp_calc_dynamic_acc_mode.getBool())
```

Comment:
Not using { curly brackets } for a non-one-liner inside an if-statement is very bad style

Reply:
I didn't even notice that there are no {}